### PR TITLE
single line disability

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Testing done
Tested on mobile and desktop

## Screenshots
disabilityRating
mobile

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/components/NameTag.jsx [hunk 1]
```diff
@@ -14,22 +14,27 @@ const DisabilityRatingContent = ({ rating }) => {
...
     'line-height--3',
+    'display--flex',
   ]);
 
   const disabilityRatingClassesMedium = prefixUtilityClasses(
     ['margin-top--1', 'text-align--left'],
     'medium',
   );
 
+  const dtRatingClasses = prefixUtilityClasses(['margin-right--0.5']);
```
Comment (erikphansen) [inline src/applications/personalization/components/NameTag.jsx:25]: should be `0p5`

**Review by erikphansen — APPROVED** [review]
Thanks for the screenshot. Does extra margin/padding need to be added on the right side of the `dt` element? Is the white space after the `:` lost because of flexbox?

**Comment by tmitchellgcio** [comment]
> Thanks for the screenshot. Does extra margin/padding need to be added on the right side of the `dt` element? Is the white space after the `:` lost because of flexbox?

yeah unfortunately the space is lost.

File: src/applications/personalization/components/NameTag.jsx (hunk 1 above)
Comment (erikphansen) [reply src/applications/personalization/components/NameTag.jsx:25]: `vads-u-margin-right--0.5` is not a class in the CSS, but `vads-u-margin-right--0p5` is https://design.va.gov/utilities/margins#margin-right

**Pull request merged**